arrows/ffmpeg/CMakeLists.txt

```diff
@@ -34,16 +34,34 @@ kwiver_add_library( kwiver_algo_ffmpeg
   ${ffmpeg_headers_public}
   ${ffmpeg_sources}
   )
+
+if(fletch_ENABLED_ZLib)
```

Comment:
Based on my understanding FFmpeg requires [zlib](https://github.com/Kitware/fletch/blob/master/CMake/External_FFmpeg.cmake#L70) in fletch and we use the system zlib to build it. When a wheel is build for unix system, it needs to follow [manylinux policy](https://www.python.org/dev/peps/pep-0599/#the-manylinux2014-policy). If the wheel has shared objects then those shared objects are only allowed to link against a white list of libraries present in list 2. Since zlib is not part of that list rely on fletch for fulfilling that dependency and explicitly point to it.

Reply:
Should this also be taken care of in a future PR?